lostfiles.py

```diff
@@ -116,11 +227,31 @@ def parse_args() -> argparse.Namespace:
     )
     return parser.parse_args()
 
+def installed_packages():
+    for pkg, dirs in PKG_PATHS.items():
+        if package_exist(pkg):
+            for dir in dirs:
+                for file in glob(dir):
+                    WHITELIST.update({file})
+
+    if package_exist("app-office/libreoffice") or package_exist("app-office/libreoffice-bin"):
+        WHITELIST.update({"/usr/lib64/libreoffice/program/resource/common/fonts/.uuid"})
+        WHITELIST.update({"/usr/lib64/libreoffice/share/fonts/truetype/.uuid"})
+
+    if check_process("systemd"):
```

Comment:
I think it is better to check if `sys-apps/systemd` is installed instead of relying on the running process, because the current implementation of `check_process()` would require `psutil` as dependency which doesn't justify for this case

Reply:
I have several servers, all of them I have installed systemd but not all I have in boot with init=/usr/lib/systemd/systemd, and
it changes which files have to be kept
That is why I search if the systemd (pid 1) is started

By the way, which ide do you use for python?

Reply:
I see, I will check if there is a way without using `psutil`, if not I'm fine with having it that way.

I'm using PyCharm for python most of the time

Reply:
Ok, it can only be verified by the pid 1 or by the file /proc/cmdline with the variable to init=/usr/lib/systemd/systemd
But it may be that in virtual systems /proc/ is not accessible

Reply:
If you want to rely on psutil, please add it as dependency to `setup.py`, something like
```
    setup(
       ...
       install_requires=['psutil']
       ...
    )
```
See https://python-packaging.readthedocs.io/en/latest/dependencies.html